repos.yaml

```diff
@@ -1282,3 +1282,8 @@ repositories:
     maintainers:
       - name: innovation4security
         email: security.group@bbva.com
+  - name: garnercorp
+    url: https://garnercorp.github.io/charts
+    maintainers:
+      - name: Garner
+        email: "garnercorp@users.noreply.github.com"
```

Comment:
The email used here needs to be a real one that we can use to contact you in case of an issue. Can you please provide a working email address.